lib/portlayer/portlayer.go

```diff
@@ -58,6 +61,30 @@ func Init(ctx context.Context, sess *session.Session) error {
 
 	// Grab the storage layer config blobs from extra config
 	extraconfig.Decode(source, &storage.Config)
+
+	logcfg := viclog.NewLoggingConfig()
```

Comment:
This should not be here. Logging setup should be part of a binary's setup, not at the library level.